core/workflow/iteratorrole.go

```diff
@@ -271,3 +271,11 @@ func (i *iteratorRole) SetRuntimeVars(kv map[string]string) {
 	}
 	i.template.SetRuntimeVars(kv)
 }
+
+func (i *iteratorRole) GetTaskPID() string {
+	if i == nil {
+		return ""
+	}
+	pid, _ := i.template.GetUserVars().Get("taskPID")
+	return pid
+}
```

Comment:
I'm not sure about this. An `iteratorRole` aggregates multiple tasks, it cannot have a PID.
Shouldn't `GetTaskPID()` only be a method on `taskRole`?